Microsoft.Toolkit.Uwp.UI.Controls.Markdown/Render/MarkdownRenderer.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Render
     /// <summary>
     /// Generates Framework Elements for the UWP Markdown Textblock.
     /// </summary>
-    public partial class MarkdownRenderer : MarkdownRendererBase
+    internal partial class MarkdownRenderer : MarkdownRendererBase
```

Comment:
Yeah, we should still expose the Renderer as part of the Control package as it allows customization of the control.

We just don't want the base parse itself to be exposed. Or are these two things more tightly coupled than we thought?

Reply:
I was having trouble keeping it public without having to make every parser class public. I have something I can try.

Reply:
The other option we have is to mark the parser bit `Obsolete` if we need to leave things public? At least by moving it out of it's own package it's a stronger signal until we can refactor the `MarkdownTextBlock` in the future with #3200.

Reply:
I went a head and marked most of the Parsing code as `Obsolete`.